code/game/objects/effects/decals/misc.dm

```diff
@@ -121,4 +238,118 @@
 	mouse_opacity = 0
 
 /obj/effect/decal/riverdecal/edge
-	icon_state = "riverdecaledge"
+	icon_state = "riverdecal_edge"
+
+/obj/effect/decal/corsat/symbol
+	icon = 'icons/effects/warning_stripes.dmi'
+	color = "#9c7f42"
+
+/obj/effect/decal/corsat/symbol/omega
+	icon_state = "omega"
+	color = "#534563"
+
+/obj/effect/decal/corsat/symbol/theta
+	icon_state = "theta"
+	color = "#549c42"
+
+/obj/effect/decal/corsat/symbol/gamma
+	icon_state = "gamma"
+	color = "#5d6384"
+
+/obj/effect/decal/corsat/symbol/sigma
+	icon_state = "sigma"
+
+
+/obj/effect/decal/grassdecal
+	name = "grass"
+	density = FALSE
+	anchored = TRUE
+	icon = 'icons/turf/ground_map.dmi'
+	icon_state = "grassdecal_edge"
+	mouse_opacity = 0
+
+
+/obj/effect/decal/grassdecal/corner
+	icon_state = "grassdecal_corner"
+
+/obj/effect/decal/grassdecal/corner2
+	icon_state = "grassdecal_corner2"
+
+/obj/effect/decal/grassdecal/center
+	icon_state = "grassdecal_center"
+
+/obj/effect/decal/lvsanddecal
+	name = "dirt"
+	gender = PLURAL
+	density = FALSE
+	anchored = TRUE
+	icon = 'icons/turf/ground_map.dmi'
+	icon_state = "lvsanddecal_edge"
+	mouse_opacity = 0
```

Comment:
```suggestion
	icon = 'icons/turf/ground_map.dmi'
	icon_state = "lvsanddecal_edge"
```

Reply:
They are copy pastes of some of the other decals, I just wasn't paying attention to the extra vars.